pipeline/measuring.py

```diff
@@ -244,7 +245,7 @@ def run(self, *args, **kwargs):
                 if m.background != 0 and m.background_err > 0.1:
                     norm_data = (c.sub_nandata - m.background) / m.background_err  # normalize
                 else:
-                    _logger.warning(f'Background mean= {m.background}, std= {m.background_err}, normalization skipped!')
+                    warnings.warn(f'Background mean= {m.background}, std= {m.background_err}, normalization skipped!')
```

Comment:
Logger warning is a problem: we can't control it using python's built in warning filters. I think we should probably move all the logger warnings into this format so I can catch them into the datastore error/warning handling code. That's my next PR.

Reply:
Oh, geez, what a can of worms.  I'd not used the warnings module before, and wondered why there are two different solutions for the same thing.  Came across some forum posts, and it looks like it's just kind of a mess.  My instinct would be that since we use logging for informational messages and for errors, it's weird to use something else just for a single level of message (i.e. warnings).  But, I'm not sure I understand what you mean about catching warnings; does the warnings module raise something like exceptions?

Reply:
Looking at : https://docs.python.org/3/howto/logging.html#when-to-use-logging

It says you should use logger.warning() if "there is nothing the client application can do about the situation, but the event should still be noted"

It says you should use the warnings module if "in library code if the issue is avoidable and the client application should be modified to eliminate the warning."

I suspect that our warnings fall into both of these categories.  But, it would seem to me that the latter case should actually raise an exception, not just issue a warning.

There seem to be two semantically different things here that are both called "warning".

What is it that you want to do with warnings that you can't do with logging?